.github/workflows/test.yml

```
@@ -11,16 +11,19 @@ on:
       - '**.py'
 
 jobs:
-  test-cpython:
-    name: Test CPython ${{ matrix.python_version }} on ${{ matrix.os }}
+  test:
     runs-on: ${{ matrix.os }}
```

how about 
```suggestion
    runs-on: ${{ matrix.os }}-latest
```
and drop that suffix from other places?